### PR TITLE
add iot protocol client cli candidate named toolkit 

### PR DESCRIPTION
**Pull request by sanshengshui**
**Toolkit** is a client command line tool that supports multiple IoT protocols. for example, MQTT and COAP protocols,

Provide a user optimized client tool for IoT developers and learners

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      websiteUrl = "https://github.com/IoT-Technology/IoT-Toolkit"
+    ).insert()
+  }
+
+  @ChangeSet(
+    order = "003",
+    id = "003-add_toolkit_0.4.0_as_default",
+    author = "iot-technology"
+  )
+  def migration003(implicit db: MongoDatabase) =
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala:32]:
```suggestion
  @ChangeSet(
    order = "002",
    id = "002-add_toolkit_0.4.0_as_default",
    author = "iot-technology"
  )
  def migration002(implicit db: MongoDatabase) =
```

**Review by eddumelendez — DISMISSED** [review]
just a suggestion. LGTM

File: src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala (hunk 1 above)
Comment (sanshengshui) [reply src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala:32]: :heartpulse:   thank for your suggestion

sdkman will get better

**Comment by sanshengshui** [comment]
please help CR :pray:

**Comment by sanshengshui** [comment]
please help me PR!

File: src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          Version(
+            "toolkit",
+            "0.4.0",
+            s"https://github.com/IoT-Technology/IoT-Toolkit/releases/download/0.4.0/$binary",
+            platform
+          )
+      }
+      .toList
+      .validate()
+      .insert()
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala:48]:
```suggestion
      .insert()
      .asCandidateDefault()
```

**Review by eddumelendez — APPROVED** [review]

File: src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+@ChangeLog(order = "073")
+class ToolkitMigrations {
+
+  val CandidateName = "toolkit"
+
+  @ChangeSet(
+    order = "001",
+    id = "001-add_toolkit_migrations_candidate",
+    author = "iot-technology"
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala:14]:
```suggestion
    author = "sanshengshui"
```

File: src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        "Toolkit is a client command line tool that supports multiple IoT protocols. for example, MQTT and COAP protocols" +
+          "Provide a user optimized client tool for IoT developers and learners",
+      websiteUrl = "https://github.com/IoT-Technology/IoT-Toolkit"
+    ).insert()
+  }
+
+  @ChangeSet(
+    order = "002",
+    id = "002-add_toolkit_0.4.0_as_default",
+    author = "iot-technology"
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala:30]:
```suggestion
    author = "sanshengshui"
```

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
thanks for the contribution @sanshengshui ! and sorry for the delay on this. the candidate should be available soon

**Comment by sanshengshui** [comment]
respect, Thanks!

**Pull request merged**